frontend/tests/integration/components/transportation/study-area-map-test.js

```diff
@@ -80,4 +80,35 @@ module('Integration | Component | transportation/study-area-map', function(hooks
 
     assert.ok(find("[data-test-popup='census-tract']"));
   });
+
+  test('it selected features on click', async function(assert) {
+    // If a project exists with a transportation analysis
+    const project = server.create('project');
+    this.model = await this.owner.lookup('service:store')
+      .findRecord('project', project.id, { include: 'transportation-analysis'});
+
+    let events = {};
+    const geoid = 'test';
+    this.map = {
+      ...DEFAULT_MAPBOX_GL_INSTANCE,
+      queryRenderedFeatures() {
+        return [{
+          type: 'Feature',
+          properties: {
+            geoid: geoid,
+          },
+        }]
+      },
+      on: (event, action) => {
+        events[event] = action;
+      }
+    };
+
+    await render(hbs`{{transportation/study-area-map analysis=model.transportationAnalysis}}`);
+    events['click']({point: 'test'});
+    await settled();
+
+    const updatedStudySelection = await this.get('model.transportationAnalysis.jtwStudySelection');
+    assert.ok(updatedStudySelection.includes(geoid));
```

Comment:
I think I follow but I'm confused by the difference between the value of `geoid` variable, and the value `test` that's being passed in the artificial click event.

Reply:
oh i was just passing in 'test' as the point b/c the actual value of the point is meaningless, since queryRenderedFeatures() is mocked to return the feature with geoid = geoid and wanted to make that clear.

However, if it is adding confusion i will update the test !